common/source/docs/common-telemetry-landingpage.rst

```diff
@@ -36,6 +36,7 @@ Long Range
     Robsense SwarmLink <common-telemetry-robsense-swarmlink>
     SPL Satellite Telemetry <https://discuss.ardupilot.org/t/stretching-comm-links-from-indoors-to-the-globe/45896>
     UAVCast 3G/4G Cellular <common-uavcast-telemetry>
+	LTM telemetry <common-ltm-telemetry>	
```

Comment:
little white space issue and it's not in alphabetical order in the list.. but we should probably move it up..

Reply:
will merge and immediately hot patch